disco.py

```diff
@@ -1,9 +1,16 @@
 # %%
+# !! {"metadata": {
```

Comment:
Does `ipynb-py-convert` handle this already? Or is this only handled by some changes you've made to the converter?

Reply:
The current conversion tool does not handle the meta data, i edited the `__main__.py` file for the converter located in site packages after pip installing, modified code is here https://github.com/MSFTserver/ipynb-py-convert/blob/master/ipynb_py_convert/__main__.py
I will also be looking into a way to just streamline this process and include it in the repo files as a simply .py script run against the file